ruma-events/tests/room_message.rs

```diff
@@ -108,13 +112,13 @@ fn plain_text_content_serialization() {
 
 #[test]
 fn relates_to_content_serialization() {
-    let message_event_content = MessageEventContent::Text(
-        assign!(TextMessageEventContent::plain("> <@test:example.com> test\n\ntest reply"), {
-            relates_to: Some(Relation::Reply {
-                in_reply_to: InReplyTo { event_id: event_id!("$15827405538098VGFWH:example.com") },
-            }),
+    // TODO: this should fail once relates_to + new_content are merged, then fix test
```

Comment:
Why would this fail?

Reply:
I think I read the dependency between `relates_to` and `new_content` wrong I was thinking it meant that if an event has a `relates_to` field it needs a `new_content` but I think I see it is the opposite, if an event has `new_content` it has to have a `relates_to` field. I'll just remove the TODO.

Reply:
Actually it's different still: Only if `relates_to` is present and contains `Relation::Replacement(_)`, *then* there has to be a `new_content`. If there is a `new_content` without that I figure we should just ignore it. But in any case, merging those two is another issue 🙂